src/SoftRobots/component/constraint/model/CableModel.h

```diff
@@ -118,6 +132,18 @@ class SOFA_SOFTROBOTS_API CableModel : virtual public SoftRobotsConstraint<DataT
     Data<Real>                  d_cableInitialLength;
     Data<Real>                  d_cableLength;
 
+    Data<sofa::helper::OptionsGroup>        d_method;
+    Data<VecCoord>              d_centers;
+    bool                        m_hasCenters;
+    Data<type::vector<Real>>    d_radii;
+    SingleLink<CableModel<DataTypes>, sofa::core::topology::BaseMeshTopology, BaseLink::FLAG_STOREPATH | BaseLink::FLAG_STRONGLINK> l_surfaceTopology;
+    type::vector<SetIndexArray> m_areaIndices;
+    type::vector<type::vector<Real>> m_ratios;
+
+    type::vector<Real> m_alpha;
+    type::vector<Real> m_beta;
```

Comment:
Could you add a description of the variable as alpha and beta are not very explicit names (or find a better name)